gobcore/message_broker/notifications.py

```diff
@@ -84,6 +84,8 @@ def get_notification(msg):
     """
     if msg.get(NOTIFICATION_TYPE) == EventNotification.type:
         return EventNotification.from_msg(msg)
+    elif msg.get(NOTIFICATION_TYPE) == DumpNotification.type:
+        return DumpNotification.from_msg(msg)
```

Comment:
Why do you make a new notification type? Multiple processes should be able to event notifications, meaning that they are interested in knowing about any new events.